WordPress/Classes/Utility/Media/ImageDownloader+Gravatar.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+extension ImageDownloader {
+
+    func downloadGravatarImage(with email: String, completion: @escaping (UIImage?) -> Void) {
+
+        guard let url = Gravatar.gravatarUrl(for: email) else {
+            completion(nil)
+            return
+        }
+
+        if let cachedImage = ImageCache.shared.getImage(forKey: url.absoluteString) {
```

Comment:
It's probably not ideal to cache avatars in memory without providing a time-to-live interval that roughly matches the HTTP cache control headers (around 5 minutes?). If you change the outside of the app, it won't update until the memory cache clears, which may need to happen while the app is running.

Reply:
Good point! Will address this in a separate PR